### PR TITLE
All breweries#8

### PR DESCRIPTION
**Pull request by KathleenYruegas**
__x__ Wrote Tests
__x__ Implemented
__x__ Reviewed

# Neccesary checkmarks:
- [x] All Tests are Passing

- [x] The code will run locally

## Type of change
- [x] New feature
- [] Bug Fix

# Implements/Fixes:
* description: Creates endpoint for breweries index. Creates serializer to display just the names for the index call.
closes story #8 

# Check the correct boxes

- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please include a link to a gif of how you feel about this branch:
https://media.giphy.com/media/LTYT5GTIiAMBa/giphy.gif

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   namespace :api do
     namespace :v1 do
-      get '/food-trucks', to: 'food_trucks#index'
+      get '/food-trucks', to: 'food_trucks#index' 
+      resources :breweries, only: [:index]
```
Comment (abdullaqudrat) [inline config/routes.rb]: We should keep defining routes consistent. Do you prefer the explicit way, or the "rails" way?

Comment (KathleenYruegas) [reply config/routes.rb]: I think if we're going to have full CRUD, the rails way is much cleaner.  What do you think?

Comment (KathleenYruegas) [reply config/routes.rb]: I agree, it should be consistent though.

Comment (KathleenYruegas) [reply config/routes.rb]: Let me change  line 5 in my branch to match and then I'll repush my code

Comment (KathleenYruegas) [reply config/routes.rb]: Actually, because of that hyphen in the food-trucks url, I can't do the 'rails' resources for food_trucks.

Comment (abdullaqudrat) [reply config/routes.rb]: We could change the url to underscores and go the rails way.

Comment (KathleenYruegas) [reply config/routes.rb]: Perfect!  I can do a fix on a new branch after this one is set up. More of a refactor branch.

**Pull request merged**